paradise.dme

```diff
@@ -1899,6 +1900,7 @@
 #include "code\modules\mob\living\simple_animal\hostile\pirate.dm"
 #include "code\modules\mob\living\simple_animal\hostile\russian.dm"
 #include "code\modules\mob\living\simple_animal\hostile\spaceworms.dm"
+#include "code\modules\mob\living\simple_animal\hostile\spiderman.dm"
```

Comment:
Hmmmm? Is this supposed to be including itself in the .dme twice?

Reply:
Good catch